pcdsdevices/pseudopos.py

```diff
@@ -785,19 +785,26 @@ def inverse(self, real_pos):
         return self.PseudoPosition(delay=delay_value + self.user_offset.get())
 
     def set_current_position(self, position):
```

Comment:
What if instead of the new method, we changed this to:
```suggestion
    def set_current_position(self, position: float = 0.0):
```

Just a thought - I also feel like the `set_zero` method is unnecessary.

Reply:
I'd rather have the set_current_position not have a default value. This could lead to bad surprises with the motor position is someones runs add accidentally without a positional argument.

I'll remove the set_zero position for now, it's easy enough to add in the future if we feel we need it